### PR TITLE
fix: remove redundant param from endpoint

### PR DESCRIPTION
**Pull request by AlexanderLukin**
The `v1/modules/:module_id/keys` endpoint can accept only two query params (`used` and `operatorIndex`) that it uses to filter keys from the output. But previously public interface of this endpoint incorrectly stated that it also can accept the `moduleAddresses` param. It doesn't make sense, because module ID is specified in the `:module_id` URL param for this endpoint.

Now the `moduleAddresses` search param has been removed from the public interface of the endpoint.

### CODE REVIEW

File: src/http/sr-modules-keys/sr-modules-keys.controller.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ import * as JSONStream from 'jsonstream';
 import type { FastifyReply } from 'fastify';
 import { ModuleIdPipe } from '../common/pipeline/module-id-pipe';
 
+class ReducedKeyQuery extends PickType(KeyQuery, ['used', 'operatorIndex']) {}
```
Comment (Amuhar) [inline src/http/sr-modules-keys/sr-modules-keys.controller.ts:29]: Like

File: src/http/sr-modules-keys/sr-modules-keys.service.ts [hunk 2]
```diff
@@ -40,7 +41,7 @@ export class SRModulesKeysService {
 
   async getModuleKeys(
     moduleId: string | number,
-    filters: KeyQuery,
+    filters: KeysFilter,
```
Comment (Amuhar) [inline src/http/sr-modules-keys/sr-modules-keys.service.ts:44]: lets use types from common/entities better

Comment (AlexanderLukin) [reply src/http/sr-modules-keys/sr-modules-keys.service.ts]: OK, I agree. I've just updated this type.

**Comment by AlexanderLukin** [comment]
UPD.

Remove unnecessary `moduleAddresses` query param from all endpoints where it was declared, but not used. These endpoints are:
`/v1/modules/keys`
`/v1/modules/:module_id/keys`
`v1/modules/:module_id/operators/keys`

**Review by Amuhar — APPROVED** [review]

**Review by Amuhar — APPROVED** [review]

**Review by Amuhar — APPROVED** [review]

**Pull request merged**